Support downloading Android emulator system image on Linux aarch64

rmartin16 opened the pull request:
## Changes
- Adds support to create an AVD on Linux for aarch64

## Notes
- This was the final major issue I had from my experimentation in #449
  - There's still the need to add `android.aapt2FromMavenOverride` to the `gradle.properties` file....but that's manageable as a manual thing....especially since you've still gotta get a working Android SDK first
- The changenote is `misc` so as not to give the impression that Android on Linux for aarch64 works without quite a bit of prerequisite work

## PR Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [X] All new features have been tested
- [X] All new features have been documented
- [X] I have read the **CONTRIBUTING.md** file
- [X] I will abide by the code of conduct

freakboy3742 reviewed (approved):
Seems like a fairly straightforward change; I'll have to take your word that it works. 

To clarify what's happened here - is this a case of something changing on the Java/Android side, or was this just an easy-win configuration change that we missed when we added the architecture checks?

rmartin16 commented:
> To clarify what's happened here - is this a case of something changing on the Java/Android side, or was this just an easy-win configuration change that we missed when we added the architecture checks?

More of an easy-change config win that I didn't include in #1360. At the time, I was still coming to grips with more of my current understanding of the Android SDK.....so, I left out this change at the time. Today, though, I added building an Android app to my "proto-buildbot" [experiment](https://github.com/rmartin16/briefcase-arch-test/actions/runs/6724923700/job/18278223693) and realized I felt more comfortable about adding this support for Linux on aarch64 now.
